### PR TITLE
installing hmtnote

### PR DESCRIPTION
**Pull request by projectoriented**
### This PR adds | fixes:

- 

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by projectoriented** [comment]
All red marks are caused by ` Invalid label key. ` for respective Dockerfile.

**Comment by projectoriented** [comment]
Docker's updated recommendations on labels:

`Label keys should begin and end with a lower-case letter and should only contain lower-case alphanumeric characters, the period character (.), and the hyphen character (-). Consecutive periods or hyphens are not allowed.`

https://docs.docker.com/config/labels-custom-metadata/

File: lib/MIP/Program/HmtNote.pm [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    my $filehandle;
+    my $infile_path;
+    my $offline;
+    my $outfile_path;
+    my $stderrfile_path;
+    my $stderrfile_path_append;
+    my $stdinfile_path;
+    my $stdoutfile_path;
+
+    ## Default(s)
```
Comment (jemten) [inline lib/MIP/Program/HmtNote.pm:59]:
```suggestion
```

File: lib/MIP/Recipes/Analysis/Mt_annotation.pm [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+## MIPs lib/
+use MIP::Constants qw{ $LOG_NAME $NEWLINE $UNDERSCORE };
+
+BEGIN {
+
+    require Exporter;
+    use base qw{ Exporter };
+
+    # Set the version for version checking
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Mt_annotation.pm:27]:
```suggestion
```
Looks like the template has not been updated in a while

File: lib/MIP/Recipes/Analysis/Mt_annotation.pm [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+    # Functions and variables which can be optionally exported
+    our @EXPORT_OK = qw{ analysis_mt_annotation };
+
+}
+
+sub analysis_mt_annotation {
+
+## Function : Annotate your mitochondrial variants here, add'l info field
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Mt_annotation.pm:35]: *add'l* 
Is that some american slang that I dont't know? 😄

**Review by jemten — COMMENTED** [review]
Looks good Mei! 
I left some very minor comments on the formatting. Maybe you want to review it also @henrikstranneheim, @ramprasadn, since me and Mei wrote large parts of it together.

File: lib/MIP/Recipes/Analysis/Mt_annotation.pm (hunk 3 above)
Comment (projectoriented) [reply lib/MIP/Recipes/Analysis/Mt_annotation.pm:35]: it's short for additional 😃

Comment (jemten) [reply lib/MIP/Recipes/Analysis/Mt_annotation.pm:34]: I've learned something today :)

**Review by jemten — APPROVED** [review]
Alright I'm approving this :)

**Comment by henrikstranneheim** [comment]
Well done Mei!

**Pull request merged**